### ISSUE TITLE
[CI] 6.x packaging tests failing on CentOS

### ISSUE THREAD

**Issue by DaveCTurner** [elastic/elasticsearch#27370]
The [6.x packaging tests](https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+packaging-tests/) are consistently failing on CentOS. The first bad build was [292 at 2017-11-10T02:18Z](https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+packaging-tests/292/) with these lines:

```
not ok 26 [TAR PLUGINS] install ingest-attachment plugin
not ok 26 [TAR PLUGINS] install ingest-attachment plugin
# (from function `assert_file_exist' in file /project/build/bats/utils/utils.bash, line 146,
# (from function `assert_file_exist' in file /project/build/bats/utils/utils.bash, line 146,
#  from function `assert_module_or_plugin_file' in file /project/build/bats/utils/utils.bash, line 220,
#  from function `assert_module_or_plugin_file' in file /project/build/bats/utils/utils.bash, line 220,
#  from function `install_and_check_plugin' in file /project/build/bats/utils/plugins.bash, line 175,
#  from function `install_and_check_plugin' in file /project/build/bats/utils/plugins.bash, line 175,
#  in test file /project/build/bats/tests/25_tar_plugins.bats, line 212)
#  in test file /project/build/bats/tests/25_tar_plugins.bats, line 212)
#   `install_and_check_plugin ingest attachment bcprov-jdk15on-*.jar tika-core-*.jar pdfbox-*.jar poi-3.15.jar poi-ooxml-3.15.jar poi-ooxml-schemas-*.jar poi-scratchpad-*.jar' failed
#   `install_and_check_plugin ingest attachment bcprov-jdk15on-*.jar tika-core-*.jar pdfbox-*.jar poi-3.15.jar poi-ooxml-3.15.jar poi-ooxml-schemas-*.jar poi-scratchpad-*.jar' failed
# -> Downloading file:///project/build/bats/archives/ingest-attachment-6.1.0-SNAPSHOT.zip
# -> Downloading file:///project/build/bats/archives/ingest-attachment-6.1.0-SNAPSHOT.zip
# 
```

This build includes commits 15a890b71f93206bc10fdd7e1662fd7f9a0edaae, 0a50fca0ee1fca0c84aeba5815e3e3f98c0aa58e and 0293c405df0790ff257db7871cd49ebb1dc01a8b over the previous good build and I vaguely suspect 0a50fca0ee1fca0c84aeba5815e3e3f98c0aa58e.

**Comment by ywelsch** [comment]
That failure should be fixed by d4b7f7a58d. I think the new failures are different ones.

**Comment by DaveCTurner** [comment]
Apologies, you're right. It's now failing for a different reason - opened #27372 to avoid confusion.
